lib/utils.js

```diff
@@ -29,18 +29,53 @@ const path = require('path');
 const electron = require('electron');
 const app = electron.app || electron.remote.app;
 
+// define the defaultDataPath and current directory
+const defaultBasePath = 'storage';
+let currentDirectory = '';
+
+/**
+ * @summary Parse directory to avoid parent directories
+ * @function
+ * @private
+ *
+ * @returns {Strings} parsed user data path without parent directories
+ */
+function getDirectory(directory){
```

Comment:
As a consequence, `path.join(app.getPath('userData'), 'storage')` should be the initial value of `currentDirectory`.